vm/src/sysmodule.rs

```diff
@@ -54,18 +54,23 @@ pub fn make_module(vm: &VirtualMachine, module: PyObjectRef, builtins: PyObjectR
         "cache_tag" => ctx.new_str("rustpython-01".to_string()),
     });
 
-    let path_list = match env::var_os("PYTHONPATH") {
-        Some(paths) => env::split_paths(&paths)
-            .map(|path| {
-                ctx.new_str(
-                    path.to_str()
-                        .expect("PYTHONPATH isn't valid unicode")
-                        .to_string(),
-                )
-            })
-            .collect(),
-        None => vec![],
-    };
+    fn get_paths(env_name: &str) -> impl Iterator<Item = std::path::PathBuf> {
+        match env::var_os(env_name) {
+            Some(paths) => env::split_paths(&paths),
+            None => env::split_paths(""),
+        }
+    }
+
+    let path_list = get_paths("RUSTPYTHONPATH")
```

Comment:
Could we figure out this path from the location of the executable? This is how cpython does it, maybe we could do the same?

Reply:
> This is how cpython does it

Based on [this code](https://github.com/python/cpython/blob/022be02dcfdfd9011415804bb4553a33fa7ec8f3/Python/pathconfig.c#L248), I think it's based on an external config option, which an environment variable effectively is. Even so, with the way that cargo works, the `Lib` directory won't always be in the same place relative to the executable.